src/Control/Eff/Cut.hs

```diff
@@ -2,6 +2,8 @@
 {-# LANGUAGE PatternGuards #-}
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
@qnikst ``TypeFamilies`` is not needed for the implementation of ``OpenUnion`` implementation (which is what gets enabled for ghc-7.6), and while the above doesn't hurt, it might be better to bound it using cpp to versions using 7.8 and above and/or the use of ``OpenUnion.Internal.OpenUnion2``.